support-frontend/assets/pages/showcase/components/hero.tsx

```diff
@@ -41,15 +47,17 @@ const getCountrySelector = (countryGroupId: CountryGroupId) => {
 	}
 };
 
-export default function Hero(props: { countryGroupId: CountryGroupId }) {
+export default function Hero(props: {
+	countryGroupId: CountryGroupId;
+}): EmotionJSX.Element {
 	const countrySelector = getCountrySelector(props.countryGroupId);
 	return (
 		<div className="showcase-hero">
 			<figure className="showcase-hero-wrapper">
 				<div className="showcase-hero-heading">
-					<h1 className="visually-hidden">Support the guardian</h1>
+					<h1 className="visually-hidden">Support the Guardian</h1>
 					<h2 className="visually-hidden">
-						Help us support investigative independent journalism
+						Help us deliver independent investigative journalism
```

Comment:
I've changed this to match what's displayed on [the support showcase page](https://support.theguardian.com/eu/support):

<img width="504" alt="image" src="https://user-images.githubusercontent.com/11380557/154264406-4f921a6a-c7ac-4c36-9ce2-1aef6b857f1a.png">